pipeline/merger/src/main/scala/weco/pipeline/merger/services/MergerManager.scala

```diff
@@ -19,7 +19,7 @@ trait MergerManager {
     val modifiedWorks = preMergeModify(works)
     if (works.size == maybeWorks.size) {
       val result = mergerRules.merge(modifiedWorks)
-      assert(result.resultWorks.size == modifiedWorks.size)
+      assert(result.resultWorks.size >= modifiedWorks.size)
```

Comment:
Possibly worth a comment explaining why this is the case? The "greater than" case isn't obvious unless you know about internal works.